### PR TITLE
ConsoleInteraction.py: Fix docstring typo

### PR DESCRIPTION
**Pull request by mbiesek**
This fixes the typo and changes
it from choose_action --> chosen_action.

Fixes https://github.com/coala/coala/issues/5201

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [x] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [x] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

File: coalib/output/ConsoleInteraction.py [hunk 1]
```diff
@@ -702,7 +702,7 @@ def try_to_apply_action(action_name,
     Try to apply the given action.
 
     :param action_name:     The name of the action.
-    :param choose_action:   The action object that will be applied.
+    :param choosen_action:  The action object that will be applied.
```
Comment (monsij) [inline coalib/output/ConsoleInteraction.py]: @mbiesek  : You made a typo indeed. Change choosen_action to chosen_action.

Comment (harshhx17) [reply coalib/output/ConsoleInteraction.py]: As monsij said please make choosen_action to chosen_action
Rest looks good to me.

**Comment by nityeshaga** [comment]
Please read our [Git basics](http://api.coala.io/en/latest/Developers/Git_Basics.html) tutorial and [Writing Good Commits](http://api.coala.io/en/latest/Developers/Writing_Good_Commits.html) tutorial @mbiesek

**Comment by monsij** [comment]
@mbiesek  : Squash all your commits into one as because even if one commit is not according to the regex, the checks would fail.

**Comment by harshhx17** [comment]
You might want to squash all the commits(have a look at the git basics) and keep the final commit message according to the guidelines on writing good commit as by Nityesh(nityeshaga) :smile:

**Comment by monsij** [comment]
@mbiesek  : Please avoid closing PRs as it uneases the process

**Pull request closed without merging**